File: tests/fixtures/php_repo/src/Support.php
<?php

namespace Demo\Support;

interface Persistable
{
    public function save(string $value): void;
}

trait Logs
{
    public function log(string $value): void
    {
        $value;
    }
}

class Helper implements Persistable
{
    public function save(string $value): void
    {
        $value;
    }

    public function format(string $value): string
    {
        return $value;
    }
}

function make_helper(): Helper
{
    return new Helper();
}

File: tests/fixtures/ruby_repo/lib/demo/support.rb
module Demo
  module Logging
    def log(value)
      value
    end
  end

  module FactoryMethods
    def build(value)
      new(value)
    end
  end

  class Helper
    def save(value)
      value
    end
  end

  class Tracing
  end
end
